include/z64scene.h

```diff
@@ -362,6 +360,11 @@ typedef struct RoomContext {
     /* 0x7A */ UNK_TYPE2 unk7A[3];
 } RoomContext; // size = 0x80
 
+typedef struct RoomList {
+    /* 0x00 */ u8 count;
+    /* 0x04 */ RomFile* romFiles; // Array of rom addresses for each room in a scene
+} RoomList;
```

Comment:
```suggestion
typedef struct RoomList {
    /* 0x0 */ u8 count;
    /* 0x4 */ RomFile* romFiles; // Array of rom addresses for each room in a scene
} RoomList; // size = 0x8
```
remove trailing 0's please, as well as add a size comment.